lessonbuilder/tool/src/webapp/js/show-page.js

```diff
@@ -3615,6 +3615,7 @@ function addShortanswer() {
 	elt = clonedAnswer.find("input");
 	elt.attr("name", elt.attr("name") + n);
 
+    clonedAnswer.attr("id", "otherShortAnswer" + n);
```

Comment:
Please, solve all spacing issues

Reply:
Sorry, I think I've since fixed the tab indentation settings for my editor. I've since pushed a revised commit.